internal/bzlmod/go_mod.bzl

```diff
@@ -201,6 +216,13 @@ def sums_from_go_mod(module_ctx, go_mod_label):
         go_mod_label.package,
         "go.sum",
     ))
+    go_sum_path = module_ctx.path(go_sum_label)
```

Comment:
I remember seeing `module_ctx.path(<some label>)` fail if the file doesn't exist, before there is even a chance to test for its existence. Did you check whether that's the case?

Reply:
If this is the case, I think it's ok to just keep it as is. It might add too much complexity to try to check if this exists and control how the failure happens.